theano/gof/tests/test_graph_opt_caching.py

```diff
@@ -6,10 +6,8 @@
 import theano
 import theano.tensor as T
 
-
-
 def test_graph_opt_caching():
-    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
+    opt_db_file = theano.config.compiledir+'/*.pkl'
     os.system('rm %s'%opt_db_file)
```

Comment:
Also, running `rm` through `os.system`. will not work on windows.
I think some functions in `shutil` may be better suited.